src/Sylius/Bundle/ThemeBundle/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Exception;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class InvalidArgumentException extends \InvalidArgumentException
```

Comment:
If we do not add anything, why we want to create our own exception class? Basic \InvalidArgumentException if enough for our needs.